ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/feemarket/ZeroBaseFeeMarket.java

```diff
@@ -38,9 +38,4 @@ public Wei computeBaseFee(
   public ValidationMode baseFeeValidationMode(final long blockNumber) {
     return ValidationMode.NONE;
   }
-
-  @Override
-  public boolean implementsDataFee() {
-    return true;
-  }
```

Comment:
Had to do this to get `retrievesValidatorsFromValidatorContract_LondonFork_ZeroBaseFee` passing. It's stolen from https://github.com/hyperledger/besu/pull/5276